CMakeLists.txt

```diff
@@ -231,12 +241,32 @@ cmake_dependent_option(
 mark_as_advanced(GOOGLE_CLOUD_CPP_ENABLE_EXAMPLES)
 
 # Add any subdirectories configured in the application.
-if (GOOGLE_CLOUD_CPP_INTERNAL_DOCFX)
-    # `docfx/` must be included before any library directories. The tool is used
-    # to generate the library documentation.
-    add_subdirectory(docfx)
-endif ()
+
+# `docfx/` must be included before any library directories. The tool is used to
+# generate the library documentation.
+add_subdirectory(docfx)
+
+# We always include the external protos. If this is not needed it immediately
+# returns.
+add_subdirectory(external/googleapis)
 add_subdirectory(google/cloud)
+
+if (GOOGLE_CLOUD_CPP_USE_INSTALLED_COMMON)
+    find_package(google_cloud_cpp_common CONFIG REQUIRED)
+    if (GOOGLE_CLOUD_CPP_ENABLE_GRPC)
+        find_package(google_cloud_cpp_grpc_utils CONFIG REQUIRED)
+        find_package(google_cloud_cpp_googleapis CONFIG REQUIRED)
+        find_package(google_cloud_cpp_iam CONFIG REQUIRED)
+        find_package(google_cloud_cpp_googleapis CONFIG REQUIRED)
```

Comment:
Repeated command

Reply:
Removed and removed an unneeded command too